fvm/evm/stdlib/contract.cdc

```diff
@@ -579,12 +674,17 @@ contract EVM {
         let addr = InternalEVM.createCadenceOwnedAccount(uuid: acc.uuid)
         acc.initAddress(addressBytes: addr)
 
-        emit CadenceOwnedAccountCreated(address: acc.address().toString())
+        emit CadenceOwnedAccountCreated(address: acc.address().toString(), uuid: acc.uuid)
         return <-acc
     }
 
     /// Runs an a RLP-encoded EVM transaction, deducts the gas fees,
     /// and deposits the gas fees into the provided coinbase address.
+    ///
+    /// @param tx: The transaction to run
+    /// @param coinbase: The address of the miner who added the block
```

Comment:
I think it's an rlp-encoded transaction. Also, I believe the `coinbase` is address where the EVM transaction fee is sent which would be analogous to receiving the block reward in a non-emulated environment.

```suggestion
    /// @param tx: The rlp-encoded transaction to run
    /// @param coinbase: The address of entity to receive the transaction fees for relaying the transaction
```